include/gui_templates.h

```diff
@@ -31,37 +31,27 @@
 #include <QFont>
 #include <QDesktopWidget>
 
+// TODO: cleanup for qt6
+#if (QT_VERSION < QT_VERSION_CHECK(6,0,0))
+	#include <QScreen>
+#else
+	#include <QGuiApplication>
+#endif
+
 namespace lmms
 {
 
 
 // return DPI-independent font-size - font with returned font-size has always
 // the same size in pixels
-template<int SIZE>
-inline QFont pointSize( QFont _f )
+template<typename T>
```

Comment:
I would remove this template as it places an unnecessary burden on the callers.

You can just declare this function as
```
inline QFont pointSize(QFont _f, float size)
```
and remove the cast in the call to `setPointSizeF`.

It's then up to the caller to put something in the function that's convertible to a `float`. And that's already the case for all clients that make use of the function. For example, instead of doing the call
```
setFont(pointSize<int>(m_interpBox->font(), 8));
```
the client would call:
```
setFont(pointSize(m_interpBox->font(), 8));
```
Or if we want to "help" the compiler a bit:
```
setFont(pointSize(m_interpBox->font(), 8.));
```

Reply:
done.